### ISSUE TITLE
force-mosaic only supports tif

### ISSUE THREAD

**Issue by maxfreu** [davidfrantz/force#222]
Hi! The .tif file extensions is hardcoded in force-mosaic. Therefore it fails for my cog files. I think the easiest fix would be to give another argument for the extension.

**Comment by davidfrantz** [comment]
or just use tif extension for your COG, because essentially, it is a tif.

**Comment by maxfreu** [comment]
True. But then it would be cool to have some hint that prints with -h :)

**Comment by maxfreu** [comment]
Oh by the way, there is a way to speed up the process significally, especially for cubes with lots of small files:
Use
`find -L "$ROUT" \( -name '*.dat' -o -name '*.tif' \) | xargs basename -a | sort | uniq > $PRODUCTS`
instead of
`find -L "$ROUT" \( -name '*.dat' -o -name '*.tif' \) -exec basename {} \; | sort | uniq > $PRODUCTS`
The problem is that `-exec` invokes basename for each file, which is slow. `xargs` comes pre-packaged with Ubuntu, so should work for all users.

**Comment by davidfrantz** [comment]
Thanks, that makes a lot of sense. Can you pull request this?

Best,
David

**Comment by davidfrantz** [comment]
It is now possible to specify the file extension: https://github.com/davidfrantz/force/commit/e2945a25e556554450bd7c69025393bbaab7fa44
